src/modules/tree.ts

```diff
@@ -315,6 +315,21 @@ export default {
     handler: './auth/whoami',
   },
   workspace: {
+    abtest: {
+      start: {
+        description: 'Start AB testing with current workspace',
+        handler: './workspace/abtest/start',
+        optionalArgs: ['weight', 'probability'],
```

Comment:
I think that there is not this optional argument `weight` anymore.

Reply:
True, thanks!